### ISSUE TITLE
fetch': key not found: :ciphers (KeyError)

### ISSUE THREAD

**Issue by scottlorimor** [guilhermesad/rspotify#135]
key not found: :ciphers

``` def index
   if !params[:artist_name].empty?
     @artists = RSpotify::Artist.search(params[:artist_name])
   else
     redirect_to root_path
   end```

  #[fetch': key not found: :ciphers (KeyError)](https://github.com/rest-client/rest-client/issues/612)

**Comment by eef** [comment]
Have you resolved this issue at all?

**Comment by jubishop** [comment]
i'm having this problem too.  it's using an old version of rest-client

**Comment by eef** [comment]
I forked it and updated the gemspec to get it working quickly.  https://github.com/eef/rspotify

**Comment by guilhermesad** [comment]
Fixed by #136
